src/styles/media.scss

```diff
@@ -0,0 +1,17 @@
+@media screen and (max-width: 1024px) {
```

Comment:
there is no need to create a separate file for these styles. it is still related to `.catalog` so please move them to `_catalog.scss` file